src/components/accordion-disclosure/accordion-disclosure.test.tsx

```diff
@@ -60,9 +60,9 @@ describe('AccordionDisclosure', () => {
 		// best effort to query for plain divs rendered by Disclosure
 		const buttons = queryAllByRole('button')
 
-		expect(buttons[0].parentElement).not.toHaveClass('isNested')
-		expect(buttons[1].parentElement).toHaveClass('isNested')
-		expect(buttons[2].parentElement).toHaveClass('isNested')
+		expect(buttons[0].parentElement).not.toHaveClass(/isNested/)
```

Comment:
These `toHaveClass` changes were needed because Vitest correctly appends hashes to class names, whereas the Jest CSS module mock did not.